include/index.h

```diff
@@ -151,6 +151,10 @@ template <typename T, typename TagT = uint32_t, typename LabelT = uint32_t> clas
 
     // Filter support search
     template <typename IndexType>
+    DISKANN_DLLEXPORT std::pair<uint32_t, uint32_t> search_with_filters(const T *query, const std::string &filter_label,
```

Comment:
what is the usage of the return value? I saw the first value of std::pair is always 0.

Reply:
first is supposed to count number of hop, currently not in use as confirmed from Ravi

Reply:
can we only return error code? since there may be more metrics in the future, it's better to move it to an object and transfer as input params.